assets/reducers/popup.js

```diff
@@ -9,7 +10,8 @@ const defaultState = {
     point: null,
     isFetchingElevation: false,
     elevation: null,
-    values: []
+    values: [],
+    region: ''
```

Comment:
So I initially followed that approach but had issues when rendering the `RegionStep` component. Since I have `region` as a required prop, the render call fails since we're defaulting the value to `null`.

Not really sure how to handle it other than an empty string for React. Any ideas?